client/src/components/TopTen/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/client';
+import { GET_TOPTEN } from '@queries';
+
+const TopTenList = styled.ul`
+  padding: 0;
+  flex-basis: 100%;
+  margin-left: 34px;
+  margin: 1rem 0 0 34px;
```

Comment:
your margin left is redundant
```suggestion
  margin: 1rem 0 0 34px;
```

Reply:
over looked that